src/r2x/utils.py

```diff
@@ -181,11 +181,38 @@ def update_dict(base_dict: dict, override_dict: ChainMap | dict | None = None) -
     return base_dict
 
 
-def read_yaml(fname: str) -> dict:
+def read_user_dict(fname: str) -> dict:
     """Load Yaml to Python dictionary."""
-    # Read configuration file
-    with open(fname) as f:
-        return yaml.safe_load(f)
+    if fname.strip().startswith("[]"):
```

Comment:
why does this also have a closing square bracket?

Reply:
I just fix it. It was a typo